src/main/java/greetbot/Parser.java

```diff
@@ -0,0 +1,52 @@
+package greetbot;
+
+/**
+ * A class which parses commands or arguments of a command.
+ */
+public class Parser {
+
+    /**
+     * Returns a string array which contains a keyword and its arguments.
+     * @param input The input from user keyboard.
+     * @return
+     */
+    public static String[] parseCommand(String input) {
+        String[] command = input.split(" ", 2);
+        command[0] = command[0].toUpperCase();
+        return command;
+    }
+
+    /**
+     * Returns a integer which indicates the position in task list.
+     * @param input The argument for mark/unmark/delete keyword.
+     * @return The position to mark/unmark/delete.
+     */
+    public static int parseMarkUnmarkDelete(String input) {
```

Comment:
Seems like a messy name. Maybe can shorten it while maintaining the same functionality?

Reply:
Second this!